app-text/qpdfview/patches/qpdfview-0.4.18.patchset

```diff
@@ -42,3 +42,156 @@ index 6363c32..dc10a8c 100644
 -- 
 2.30.0
 
+
+From c5d6a0ffa020ce7b75469ef778ef7d1f250e0245 Mon Sep 17 00:00:00 2001
+From: FuRuYa7 <mymail@mydomain.org>
+Date: Fri, 29 Oct 2021 15:03:10 +0900
+Subject: [PATCH 1/2] Fix installation path
```

Comment:
This should be defined on the qmake command line in the recipe

Reply:
I don't have the expected "qmake" skills.

Reply:
I remember getting an undefined error when I specified "$dataDir" in the qmake location. I gave up at that time.

Reply:
I try using "$appDir" in qmake. Please wait.

Reply:
As you pointed out, the translation was reflected with the following changes. You are right. 

However, the help remains in English. A help switching line is required. I uncommented it and confirmed that it was translated into Japanese. 


```
	qmake ../qpdfview.pro \
		CONFIG+="without_ps without_cups without_synctex without_dbus without_signals" \
		TARGET_INSTALL_PATH=$appsDir/qPDFView \
		PLUGIN_INSTALL_PATH=$appsDir/qPDFView/lib \
		DATA_INSTALL_PATH=$appsDir/qPDFView/data \
		MANUAL_INSTALL_PATH=$manDir

	make $jobArgs
}

INSTALL()
{
	cd haiku_build
	make install
	#Translate Help file (Please uncomment and change the language)
	#cp $appsDir/qPDFView/data/help_ja.html $appsDir/qPDFView/data/help.html
	
	mv $appsDir/qPDFView/qpdfview $appsDir/qPDFView/qPDFView
```
--
It's midnight, so that's it for today.

Reply:
I tried using "$dataDir" with qmake. 

qpdfview.pri :

```
isEmpty(DATA_INSTALL_PATH):DATA_INSTALL_PATH = $dataDir/qPDFView/data
```

With the changes below, the translation has been reflected :

```
	qmake ../qpdfview.pro \
		CONFIG+="without_ps without_cups without_synctex without_dbus without_signals" \
		TARGET_INSTALL_PATH=$appsDir/qPDFView \
		PLUGIN_INSTALL_PATH=$appsDir/qPDFView/lib \
		DATA_INSTALL_PATH=$dataDir/qPDFView/data \
		MANUAL_INSTALL_PATH=$manDir

	make $jobArgs
}

INSTALL()
{
	cd haiku_build
	make install
	#Translate Help file (Please uncomment and change the language)
	#cp $dataDir/qPDFView/data/help_ja.html $dataDir/qPDFView/data/help.html

```

After all, the help remains in English. A help switching line is required. I uncommented it and confirmed that it was translated into Japanese.

The first thing that failed seems to be that the definition of "qpdfview.pri" was missing. 

Since the files are not scattered, I think it is better to unify to "$appsDir".

Reply:
One of the two features of "qpdfview.pri" is file placement, which used "qmake". I understood that way.

Reply:
is the patch for qpdfview.pri still needed?

Reply:
It worked without the patch for qpdfview.pri, so I removed it.

Thank you.